### PR TITLE
[Merged by Bors] - ci: Try older version of node 18

### PR DESCRIPTION
**Pull request by morenol**
This is done in order to unblock CI. We need to research why on node>=18.14 we get segmention errors

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,8 +6,9 @@ on:
     branches:
       - staging
       - trying
+      - try-older-version-of-node-18
```
Comment (morenol) [inline .github/workflows/ci.yml:9]:
```suggestion
```

**Review by sehz — APPROVED** [review]
LGTM.  Nice fix

**Comment by sehz** [comment]
see: https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md#18.14.0,

**Comment by sehz** [comment]
works on Node v18.6 on local machine

**Comment by morenol** [comment]
> works on Node v18.6 on local machine

in my local machine it fails on `18.14`

**Pull request closed without merging**